warehouse/query-core/src/main/java/datawave/query/jexl/functions/GeoFunctions.java

```diff
@@ -98,7 +110,14 @@ public static boolean within_bounding_box(Object lonField, Object latField, doub
             return false;
         }
         
-        boolean lonMatch = lonValues.stream().anyMatch(lon -> lon >= minLon && lon <= maxLon);
+        // @formatter:off
+        boolean lonMatch = lonValues.stream().anyMatch(lon ->
```

Comment:
We should probably add a test for this case

Reply:
done